WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -105,7 +105,7 @@ public LoadWalletViewModel(WalletManagerViewModel owner, LoadWalletType loadWall
 				{
 					var ofd = new OpenFileDialog
 					{
-						AllowMultiple = false,
+						AllowMultiple = true,
```

Comment:
Why?

Reply:
I was testing if that scenario worked, and accidentally committed it. Revoked.